MonstieWash/Assets/Scripts/Tool.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "Tool", menuName = "ScriptableObjects/Tool")]
+public class Tool : ScriptableObject
+{
+    public Sprite mask;
+    [HideInInspector]
+    public float strength;
+    [Range(1, 10)] 
+    public int size = 1;
+    [HideInInspector] 
+    public byte[] maskPixels;
+
+    [SerializeField]
+    [Range(1f, 100f)] 
+    private float inputStrength = 100f;
+
+    public void Initialize()
+    {
+        strength = Mathf.Pow(1.0472f, inputStrength);
```

Comment:
1.0472 should probably be explained,  eg. scaling factor, also does inputStrength need to be adjusted

Reply:
Ah good catch, will do. I'll explain more of what each variable does in the code (the 1.0472 is just 100 to the 100th root), but for now the inputStrength is what the user inputs on a slider (0 - 100) which represents the strength/opacity of the tool and then that is scaled to make the strength adjustments seem more linear and intuitive. Then strength is the final (actual) tool strength value. These are mainly settings for designers to use when setting up tools.